modules/dnn/perf/perf_halide_net.cpp

```diff
@@ -150,6 +154,115 @@ PERF_TEST(ENet, HalidePerfTest)
         throw PerfSkipTestException();
     }
 }
+////////////////////////////////////////////////////////////////////////////////
+// OpenCL target
+////////////////////////////////////////////////////////////////////////////////
+PERF_TEST(GoogLeNet_opencl, HalidePerfTest)
+{
+    try {
+        Net net;
+        loadNet("dnn/bvlc_googlenet.caffemodel", "dnn/bvlc_googlenet.prototxt",
+                "", 227, 227, "prob", "caffe", DNN_TARGET_OPENCL, &net);
+
+        TEST_CYCLE_N(10)
+        {
+            net.forward();
+        }
+        SANITY_CHECK_NOTHING();
+    } catch (SkipTestException& e) {
+        throw PerfSkipTestException();
```

Comment:
Perhaps we should unify behavior for these exceptions (via patch into opencv repository).